src/components/movies-list/movies-list.js

```diff
@@ -3,47 +3,29 @@ import './movies-list.css';
 import MovieItem from '../movie-item';
 import ErrorBoudary from '../error-boudary';
 
-export default class MoviesList extends Component {
-  constructor() {
-    super();
-    this.data = [{
-      key: "m1",
-      title: "Bohemian Rhapsody",
-      description: "Drama Biography Music",
-      year: "2019",
-      img: "../../images/movie-cover.jpeg"
-    }, {
-      key: "m2",
-      title: "Bohemian Rhapsody1",
-      description: "Drama Biography Music",
-      year: "2019",
-      img: "../../images/movie-cover.jpeg"
-    }, {
-      key: "m3",
-      title: "Bohemian Rhapsody2",
-      description: "Drama Biography Music",
-      year: "2019",
-      img: "../../images/movie-cover.jpeg"
-    }, {
-      key: "m4",
-      title: "Bohemian Rhapsody3",
-      description: "Drama Biography Music",
-      year: "2019",
-      img: "../../images/movie-cover.jpeg"
-    }, {
-      key: "m5",
-      title: "Bohemian Rhapsody4",
-      description: "Drama Biography Music",
-      year: "2019",
-      img: "../../images/movie-cover.jpeg"
-    }];
-  }
 
+export default class MoviesList extends Component {
   render() {
-    let movieItems = this.data.map((item) => 
-      <MovieItem key={item.key} title={item.title} description={item.description} year={item.year} cover={item.img} />
-    );
-    let isDataReceived = this.data.length;
+    this.aData = this.props.moviesList;
+    let movieItems = this.props.moviesList.map((item) => {
+      return (<MovieItem 
+        key={item.id} 
+        id={item.id}
+        title={item.title} 
+        genres={item.genres} 
+        release_date={item.release_date} 
+        poster_path={item.poster_path} 
+        overview={item.overview}
+        runtime={item.runtime}
+        isOpenDropDown={item.isOpenDropDown} 
+        onOptionHandler={this.props.onOptionHandler}
+        onCloseDropDown={this.props.onCloseDropDown}
+        onSubmitForm={this.props.onSubmitForm}
+        onDeleteMovie={this.props.onDeleteMovie}
+        onOpenMovieDetail={this.props.onOpenMovieDetail}
+        />);
+    });
+    let isDataReceived = this.aData.length;
```

Comment:
I suggest to change variable name as, I assume that isDataReceived is boolean but aData.length is number